src/index.js

```diff
@@ -1 +1,45 @@
 import './style.css';
+import NewPlayer from './newplaya';
+import Api from './api';
+
+export default class Ui {
+  static render(players) {
+    const listItems = document.querySelector('.list');
+    players.forEach((player) => {
+      listItems.innerHTML += `
+      <li>
+        <p>${player.user}:</p>
+        <p> ${player.score}</p>
+      </li>
+      `;
+    });
+  }
+
+  static clearField() {
+    const listItems = document.querySelector('.list');
+    listItems.innerHTML = '';
+  }
+}
+
+async function getData() {
+  const pull = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/Ml0wfxegyFf06umHiVlI/scores/');
+
+  const data = await pull.json();
+  const dataOut = await data.result;
+  Ui.render(dataOut);
+}
```

Comment:
- [ ] Kindly use the ES6 arrow function for this method instead of the function keyword as per the project's requirements.